frontend/apps/filemanager/filemanagershortcuts.lua

```diff
@@ -6,21 +6,22 @@ local InputDialog = require("ui/widget/inputdialog")
 local Menu = require("ui/widget/menu")
 local Screen = require("device").screen
 local UIManager = require("ui/uimanager")
+local lfs = require("libs/libkoreader-lfs")
 local util = require("ffi/util")
 local _ = require("gettext")
 local T = util.template
 
 local FileManagerShortcuts = InputContainer:extend{}
 
+local function tableSortByTextProp(subject)
+    table.sort(subject, function (v1, v2)
+        return v1.text < v2.text
```

Comment:
Are you sure this isn't equivalent to table.sort without supplying a custom sorting function?

Reply:
The context is that in my private installation I have several helper methods for custom sorting operations which I can quickly use, without typing much code. But now I've pushed a sorting operation that doesn't require a local function.

Reply:
That doesn't address my question though: isn't this quite literally the default behavior?

Reply:
Okay, the answer is the table is always deeper. So carry on, but there was no hidden meaning. ;-)